wandb/sdk/internal/sender.py

```diff
@@ -606,12 +607,21 @@ def send_artifact(self, data):
 
     def send_alert(self, data):
         alert = data.alert
-        self._api.notify_scriptable_run_alert(
-            title=alert.title,
-            text=alert.text,
-            level=alert.level,
-            wait_duration=alert.wait_duration,
-        )
+        _, server_info = self._api.viewer_server_info()
```

Comment:
Really old version of local will return an empty dict for `server_info`.  We should handle that case as well.

Reply:
Done.